src/pages/index.js

```diff
@@ -1,168 +1,241 @@
+      .then(() => {
+        card.handleDeleteCard();
+        deleteCardPopup.close();
+      })
+
+      .catch((err) => {
+        console.log(err);
+      })
+      .finally(() => {
+        deleteCardPopup.renderLoading(false);
+      });
+  });
+}
 
 function handleProfileEditClick() {
   const user = userInfo.getUserInfo();
-
   userNameInput.value = user.name;
-
-  userDescriptionInput.value = user.description;
-
+  userDescriptionInput.value = user.about;
   profileEditPopup.open();
 }
 
-function handleProfileEditSubmit(inputValues) {
-  userInfo.setUserInfo(inputValues);
-  profileEditPopup.close();
+function handleLikeClick(card) {
+  if (card.isLiked) {
+    api
+      .removeCardLikes(card.cardId)
+      .then((res) => {
+        card.updateLikes(res.isLiked);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  } else {
+    api
+      .addCardLikes(card.cardId)
+      .then((res) => {
+        card.updateLikes(res.isLiked);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
 }
 
-/* ----------------------------- add card modal ----------------------------- */
-
-const addCardButton = document.querySelector(".profile__add-button");
-
-const addCardPopup = new PopupWithForm("#card-edit-modal", handleAddCardSubmit);
-
-addCardPopup.setEventListeners();
-
-addCardButton.addEventListener("click", () => {
-  cardAddFormValidator.resetValidation;
-  cardAddFormValidator.toggleButtonState();
-  addCardPopup.open();
-});
-
-function handleAddCardSubmit(inputValues) {
-  const { title, link } = inputValues;
-  const newCardData = {
-    name: title,
-    link: link,
-  };
-  const newCard = createCard(newCardData);
-  section.addItem(newCard);
-  addCardPopup.close();
+function handleCardClick(cardData) {
+  previewImagePopup.open(cardData);
 }
 
 /* -------------------------------------------------------------------------- */
-
 /*                               FORM VALIDATION                              */
-
 /* -------------------------------------------------------------------------- */
-
+const aviFormValidator = new FormValidator(formValidatorConfig, aviEditForm);
+const editProfileFormValidator = new FormValidator(
+  formValidatorConfig,
+  profileEditForm
+);
 const cardAddFormValidator = new FormValidator(
   formValidatorConfig,
-
   cardAddForm
 );
-
+editProfileFormValidator.enableValidation();
 cardAddFormValidator.enableValidation();
+aviFormValidator.enableValidation();
 
-const editProfileFormValidator = new FormValidator(
-  formValidatorConfig,
-
-  profileEditForm
-);
-
-editProfileFormValidator.enableValidation();
+/* -------------------------------------------------------------------------- */
+/*                               EVENT LISTENERS                              */
+/* -------------------------------------------------------------------------- */
+aviEditButton.addEventListener("click", () => {
+  aviFormValidator.resetValidation;
```

Comment:
`  aviFormValidator.resetValidation;
`

`resetValidation` doesn't exist


You need to call `toggleButtonState` to disable buttons 


```js
  aviFormValidator.toggleButtonState();

```